airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1077,7 +1077,7 @@
 - name: LinkedIn Pages
   sourceDefinitionId: af54297c-e8f8-4d63-a00d-a94695acc9d3
   dockerRepository: airbyte/source-linkedin-pages
-  dockerImageTag: 0.1.0
+  dockerImageTag: 1.0.0
```

Comment:
Major bump bc we changed the format of something in the spec, so old specs will be invalid 👍🏻